packages/expo-cli/src/commands/upgrade.ts

```diff
@@ -394,22 +378,34 @@ export async function upgradeAsync(
     npm: options.npm,
     yarn: options.yarn,
     log,
+    silent: true,
   });
 
   log.addNewLineIfNone();
-  log(chalk.underline.bold('Installing the expo package...'));
+  let installingPackageStep = logNewSection('Installing the expo package...');
   log.addNewLineIfNone();
-  await packageManager.addAsync(`expo@^${targetSdkVersionString}`);
```

Comment:
Maybe we should create a superset of PackageManager that implements `expo install`